### ISSUE TITLE
I complete the SN by Keras.  welcome to advice

### ISSUE THREAD

**Issue by Bingohong** [switchablenorms/Switchable-Normalization#6]
location in [SN by Keras](https://github.com/Bingohong/SwitchableNormalization-Keras)

**Comment by pluo911** [comment]
Thanks for your interest. SN benefits from adding 0.5 dropout in the last layer of hidden features, but GN and BN might not. The improvement depends on the generation error (val_err - train_err) of your problem. The training time of SN can be improved by using CUDA. We may release this implementation in the future.

**Comment by Bingohong** [comment]
thanks to apply the dropout trick. i expect the CUDA accelerated version.
